server/models/Retailer.js

```diff
@@ -51,7 +51,7 @@ const RetailerSchema = new Schema(
       required: true,
```

Comment:
Couldn't write a comment above this line so I'll just write it here. In the password section is it OK to remove the "select: false"? passport errors out in the login strategy when bcrypt tries to compare the entered password to the looked up retailer's password.

Reply:
we set this up so the database doesn't share passwords be default.  Thinking was that it would be more secure and we wouldn't have to think about it.  I thought we covered this in our passport strategy by using "password" as the second parameter in the Retailer.find() method?

Reply:
yeah, I cannot reproduce this error.  It's working for me.

Reply:
yea I had old code still on my passport i replaced the function and everything is working my bad.